### ISSUE TITLE
Tab is expanded in Makefile

### ISSUE THREAD

**Issue by foo2810** [foo2810/ad-hoc-vimconfig#6]
**Version**
f7c2817c09ade8221a47b6719aa0021aaa0050af

**Vim Vesrion**
all version

**Details**
In Makefile, tab should not be expanded. But expanded.

**Frequency**
every time

**Comment by foo2810** [comment]
Miss configuration

```vim
function! s:set_config_make()
    setlocal expandtab
    ...
```

**Comment by foo2810** [comment]
Fixed in 3a5d64a0af3fdd88fb992c3cdd55452b742539c1.
